### PR TITLE
Support alluxio.tar.gz in alluxio-emr.sh emr bootstrap

### PR DESCRIPTION
**Pull request by LuQQiu**
Support users providing tarball with name `alluxio.tar.gz` for alluxio-emr.sh

### CODE REVIEW

File: integration/emr/alluxio-emr.sh [hunk 1]
```diff
@@ -143,9 +143,12 @@ emr_install_alluxio() {
   download_file "${alluxio_tarball}"
 
   release=$(basename "${alluxio_tarball}")
-  local release_unzip
-  release_unzip="${release%%-*}" # trims everything after the first '-', ex. alluxio-foo-bar-whatever -> alluxio
-
+  local release_prefix
```
Comment (Xenorith) [inline integration/emr/alluxio-emr.sh:146]: should this be release_unzip?

File: integration/emr/alluxio-emr.sh [hunk 2]
```diff
@@ -143,9 +143,12 @@ emr_install_alluxio() {
...
+  if [[ "${release}" == *-* ]]; then
+    release_unzip="${release%%-*}" # trims everything after the first '-', ex. alluxio-foo-bar-whatever -> alluxio
```
Comment (Xenorith) [inline integration/emr/alluxio-emr.sh:148]: both of these expressions seem very fragile to me :/
the original expression of converting alluxio-* -> alluxio seems unnecessarily difficult

is it not possible to trim the ".tar.gz" suffix from the tarball and use that as the directory name that the tarball unzips to?

Comment (LuQQiu) [reply integration/emr/alluxio-emr.sh:148]: emmm... the problem is the unzipped folder from  <value>.tar.gz doesn't name <name>

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Pull request merged**